_posts/2023-02-22-user-defined-operators.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: post
+published: true
+title: 'Feature Deep-Dive: User-Defined Operators'
+date: '2023-02-22'
+author: Kamil Śliwak, Matheus Aguiar
+category: Explainers
+---
+
+The prominent new feature in [Solidity 0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19)
+is the support for defining *operators* on [user defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs).
+
+If you have not been keeping up with recent features, UDVTs are a new class of types introduced in Solidity 0.8.8.
+They provide zero-cost abstraction of an elementary value type that results in a completely new type.
+This is similar to creating an alias, but the new type is distinct from the underlying value type
```

Comment:
I'm not sure I would say that it is similar to creating an alias, because it is not ;)
Also what do you mean by "all other UDVTs derive from it"?

Reply:
> I'm not sure I would say that it is similar to creating an alias, because it is not ;)

This is another thing that was suggested to me: https://github.com/ethereum/solidity-blog/pull/202#discussion_r1085549651. I mean, I know it's not an alias, which is why I reworded the original suggestion, but still, it's not like there are similarities at all. That's all it says and the later part of the sequence points out what's not similar.

> Also what do you mean by "all other UDVTs derive from it"?

All the UDVTs that use the same underlying built-in type.

---

Anyway, I'm fine with rewording it, or even removing that part of the introduction (in my original version it wasn't there anyway). What would be your suggestion for the new wording?

Reply:
Oh, I thought "it" meant the new type...

Reply:
Changed to `derived from that underlying type`.